ui/src/containers/Resources/index.tsx

```diff
@@ -17,17 +18,20 @@ import './Resources.css';
 const Resources = () => {
   const { resources } = useMst();
 
-  const checkResources = (resources: IResource[]) => {
-    return !resources.length ? (
+  const checkResources = (resourceList: IResource[]) => {
```

Comment:
Any reason why are we changing the variable name here ??

Reply:
within function we needed `resources` store and function has local variable (function params) with same name ..

Reply:
Should we just call it as `items`

Reply:
okay will change it